openfecwebapp/templates/partials/legal-search-results-advisory-opinion.html

```diff
@@ -0,0 +1,8 @@
+<tr>
+  <td><a title="{{ advisory_opinion.description }}" href="http://saos.fec.gov/aodocs/{{ advisory_opinion.no }}.pdf#search={{ query }}">{{ advisory_opinion.no }}</a></td>
```

Comment:
Just curious why you're setting `title` to this here? It shows up as a huge block of text when hovering.